recipes/abayesqr/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: abayesqr
+  version: "1.0"
```

Comment:
Are you sure this is version 1.0? Is there any way we can get an official release? Otherwise, I guess we need to change the version number to something else.